src/ert/_c_wrappers/enkf/model_config.py

```diff
@@ -24,46 +23,42 @@ def __init__(
         history_source: Optional[HistorySourceEnum] = None,
         runpath_format_string: Optional[str] = None,
         jobname_format_string: Optional[str] = None,
+        runpath_file: str = ".ert_runpath_list",
         gen_kw_export_name: Optional[str] = None,
         obs_config_file: Optional[str] = None,
         time_map_file: Optional[str] = None,
     ):
         self.num_realizations = num_realizations
         self.refcase = refcase
         self.data_root = data_root
-
         self.history_source = (
             history_source
             if self.refcase is not None and history_source is not None
             else self.DEFAULT_HISTORY_SOURCE
         )
+        self.jobname_format_string = (
+            replace_runpath_format(jobname_format_string) or "JOB<IENS>"
+        )
+        self.runpath_format_string = (
+            replace_runpath_format(runpath_format_string) or self.DEFAULT_RUNPATH
+        )
+        self.runpath_file = runpath_file
```

Comment:
Why do we need to move the location of the runpath_file ?

Reply:
Because I think it is more logical that information about the `runpath_format_string` and the `runpath_file` are in the same place, in this case under the umbrella of `model_config`